frontend/src/views/CellGuide/components/CellGuideCard/components/MarkerGeneTables/constants.ts

```diff
@@ -2,6 +2,8 @@ export const HOMO_SAPIENS = "Homo sapiens";
 
 export const ALL_TISSUES = "All Tissues";
 
+export const TISSUE_AGNOSTIC = "Tissue Agnostic";
```

Comment:
not sure if this is the best place to throw this constant - i can move it somewhere else